app/services/slack_service.rb

```diff
@@ -2,7 +2,17 @@ class SlackService
   def initialize(client = Slack::Client.new)
     @client = client
   end
+
   def post_message(channel, message)
     @client.chat_postMessage(channel: channel, text: message, as_user: true)
   end
+
+  def post_direct_message(token, users, message)
+    members = @client.users_list(token: token)
```

Comment:
I think at this stage we're just working with the response from the API rather than `members` - maybe we can just call it `response`?